milestones/services.py

```diff
@@ -0,0 +1,44 @@
+"""
+A wrapper class around all methods exposed in api.py
```

Comment:
@efischer19  It looks like most of the contents of this file was copied from the analogous edx_proctoring/services.py file.  It would be great if we could share common code like this across the repos.

I agree that this docstring doesn't apply to the current code.